### PR TITLE
HUBDEV-1117 Code refactor workflow-approval service

### PR DESCRIPTION
**Pull request by nilanwso2**

### CODE REVIEW

File: components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java [hunk 1]
```diff
@@ -215,13 +215,14 @@ public boolean insertValidatorForSubscription(int appID, int apiID, int validato
...
-            StringBuilder query = new StringBuilder();
-            query.append("INSERT INTO subscription_validator (application_id, api_id, validator_id) VALUES ");
-            query.append("(" + appID + "," + apiID + "," + validatorID + ")");
-            st = con.createStatement();
-            st.executeUpdate(query.toString());
-
+           if(!subscriptionIsExist(appID,apiID)) {
+               con = dbUtils.getDbConnection(DataSourceNames.WSO2TELCO_DEP_DB);
+               StringBuilder query = new StringBuilder();
+               query.append("INSERT INTO subscription_validator (application_id, api_id, validator_id) VALUES ");
```
Comment (Nuwan-Walisundara) [inline components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java]: format the code ,a break them into logical blocks

File: components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java [hunk 2]
```diff
@@ -215,13 +215,14 @@ public boolean insertValidatorForSubscription(int appID, int apiID, int validato
...
+               query.append("(" + appID + "," + apiID + "," + validatorID + ")");
```
Comment (Nuwan-Walisundara) [inline components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java]: use append insted of +

File: components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java [hunk 3]
```diff
@@ -435,4 +436,114 @@ public int getOperatorIdByName(String operatorName) throws SQLException, Busines
...
+
+        Connection conn = null;
+        PreparedStatement ps = null;
+        ResultSet rs = null;
+        Boolean isExist =false;
+
+        try {
+            conn = dbUtils.getDbConnection(DataSourceNames.WSO2TELCO_DEP_DB);
+            StringBuilder query = new StringBuilder();
+            query.append("SELECT * FROM subscription_validator ");
```
Comment (Nuwan-Walisundara) [inline components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java]: specify a column

File: components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java [hunk 4]
```diff
@@ -435,4 +436,114 @@ public int getOperatorIdByName(String operatorName) throws SQLException, Busines
...
+            throws Exception, BusinessException {
+        Connection conn = null;
+        PreparedStatement ps = null;
+        ResultSet rs = null;
+        Boolean isExist =false;
+
+        try {
+            conn = dbUtils.getDbConnection(DataSourceNames.WSO2TELCO_DEP_DB);
+            StringBuilder query = new StringBuilder();
+            query.append("SELECT * FROM operatorapps ");
```
Comment (Nuwan-Walisundara) [inline components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java]: specify a column

File: components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java (hunk 1 above)
Comment (nilanwso2) [inline components/workflow/workflow-service/src/main/java/com/wso2telco/workflow/dao/WorkflowDbService.java]: what do you mean by  logical blocks.

**Pull request merged**